test/e2e/cli.go

```diff
@@ -185,6 +185,7 @@ func RunKubectl(namespace string, args ...string) (string, error) {
 		args = append(args, "--namespace", namespace)
 	}
 	stdout, stderr, err := runCli("kubectl", args)
+	//out := fmt.Sprintf("$kubectl %s \n %s", args, stdout)
```

Comment:
please remove

Reply:
for test purpose. will remove in the latest version.